service/history/replication/stream_receiver.go

```diff
@@ -87,20 +87,17 @@ func NewClusterShardKey(
 }
 
 func NewStreamReceiver(
+	logger log.Logger,
 	processToolBox ProcessToolBox,
 	taskConverter ExecutableTaskConverter,
 	clientShardKey ClusterShardKey,
 	serverShardKey ClusterShardKey,
+	highPriorityTaskTracker ExecutableTaskTracker,
+	lowPriorityTaskTracker ExecutableTaskTracker,
```

Comment:
if we move this up. Then the stream receiver should not call cancel/stop on task tracker?